### PR TITLE
Added animation slider in livreGUI

### PR DESCRIPTION
**Pull request by chevtche**

### CODE REVIEW

File: apps/livreGUI/animationController/AnimationController.cpp [hunk 1]
```diff
@@ -0,0 +1,316 @@
+/* Copyright (c) 2011-2015, EPFL/Blue Brain Project
+ *                          Ahmet Bilgili <ahmet.bilgili@epfl.ch>
```
Comment (bilgili) [inline apps/livreGUI/animationController/AnimationController.cpp]: You wrote this code

File: apps/livreGUI/animationController/AnimationController.h [hunk 2]
```diff
@@ -0,0 +1,81 @@
+/* Copyright (c) 2011-2015, EPFL/Blue Brain Project
+ *                          Ahmet Bilgili <ahmet.bilgili@epfl.ch>
```
Comment (bilgili) [inline apps/livreGUI/animationController/AnimationController.h]: ditto

File: apps/livreGUI/animationController/AnimationController.h [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+#include <zeq/hbp/vocabulary.h>
+
+#ifndef Q_MOC_RUN
+#  include <livreGUI/ui_AnimationController.h>
+#endif // Q_MOC_RUN
+
+#include <QWidget>
+
+struct AnimationControllerImpl;
```
Comment (bilgili) [inline apps/livreGUI/animationController/AnimationController.h]: what is this AnimationControllerImpl ?

File: apps/livreGUI/animationController/AnimationController.cpp (hunk 1 above)
Comment (chevtche) [inline apps/livreGUI/animationController/AnimationController.cpp]: Done

File: apps/livreGUI/animationController/AnimationController.h (hunk 2 above)
Comment (chevtche) [inline apps/livreGUI/animationController/AnimationController.h]: Done

**Comment by chevtche** [comment]
retest this please

**Comment by chevtche** [comment]
retest this please

**Pull request merged**